sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/README.md

```diff
@@ -52,12 +52,10 @@ azure:
         sign-up-or-sign-in: ${your-sign-up-or-in-user-flow}
         profile-edit: ${your-profile-edit-user-flow}      # optional
         password-reset: ${your-password-reset-user-flow}  # optional
+        signin: ${your-sign-in-user-flow} # optional  
```

Comment:
Sign in are two words, so we should change to `sign-in` here, which is also consistent with `sign-up-or-sign-in`